README.md

```diff
@@ -275,8 +275,12 @@ You can specify the browser and Node.js versions by queries (case insensitive):
   `browserslist-config-mycompany` npm package.
 * `supports es6-module`: browsers with support for specific features.
   `es6-module` here is the `feat` parameter at the URL of the [Can I Use]
-  page. A list of all available features can be found at
-  [`caniuse-lite/data/features`].
+  page. This includes browsers with full or partial support for a feature.
```

Comment:
Can you also send PR to [our website](https://browsersl.ist/) repo: https://github.com/browserslist/browsersl.ist/blob/main/client/index.html

Reply:
Will do!

Reply:
> Should I push for using fully supports as a default here to match the intent of that text? 

Yes, we can do it

Reply:
PR to browsersl.ist added: https://github.com/browserslist/browsersl.ist/pull/522